test

```diff
@@ -14,9 +14,7 @@ COVER=${COVER:-"-cover"}
 
 RACE=${RACE:-"-race"}
 
-source ./build
-
-TESTABLE="http jose key oauth2 oidc"
+TESTABLE="http jose key oauth2 oidc ."
```

Comment:
I recently switched to `PKGS=$(go list ./... | grep -v /vendor)` instead of listing packages